### PR TITLE
test(kubeflow): remove empty or not maintained tests

### PR DESCRIPTION
**Pull request by Bobgy**
Part of https://github.com/kubernetes/test-infra/issues/14343

Changes:
* clean up empty/not maintained tests
* remove trigger plugin for repos without tests

After this PR, `kubeflow/kubeflow`, `kubeflow/testing`, `kubeflow/kubeflow` are the last three repos using k8s/test-infra.

I didn't disable status reconciler, because it's OK to let it retire the contexts.

/assign @chaodaiG

### CODE REVIEW

File: config/jobs/kubeflow/kubeflow-periodics.yaml [hunk 1]
```diff
@@ -75,25 +75,3 @@ periodics:
     # TODO: use a public email group
     testgrid-alert-email: kubeflow-engineering@google.com
     testgrid-num-failures-to-alert: "3"
-#******************************************************************************
-# kubeflow/examples
```
Comment (Bobgy) [inline config/jobs/kubeflow/kubeflow-periodics.yaml:79]: not maintained

File: config/jobs/kubeflow/kubeflow-postsubmits.yaml [hunk 2]
```diff
@@ -1,30 +1,4 @@
 postsubmits:
-  kubeflow/examples:
```
Comment (Bobgy) [inline config/jobs/kubeflow/kubeflow-postsubmits.yaml:2]: not maintained

File: config/jobs/kubeflow/kubeflow-postsubmits.yaml [hunk 3]
```diff
@@ -54,20 +28,6 @@ postsubmits:
       # TODO: use a public email group
       testgrid-alert-email: kubeflow-engineering@google.com
       testgrid-num-failures-to-alert: "3"
-
-  kubeflow/website:
```
Comment (Bobgy) [inline config/jobs/kubeflow/kubeflow-postsubmits.yaml:58]: empty test

File: config/jobs/kubeflow/kubeflow-presubmits.yaml [hunk 4]
```diff
@@ -27,20 +13,6 @@ presubmits:
       testgrid-dashboards: sig-big-data
       description: Presubmit tests for Kubeflow GCP gcp-blueprints.
       testgrid-num-columns-recent: '30'
-  kubeflow/internal-acls:
```
Comment (Bobgy) [inline config/jobs/kubeflow/kubeflow-presubmits.yaml:30]: empty test

File: config/jobs/kubeflow/kubeflow-presubmits.yaml [hunk 5]
```diff
@@ -1,18 +1,4 @@
 presubmits:
-  kubeflow/examples:
```
Comment (Bobgy) [inline config/jobs/kubeflow/kubeflow-presubmits.yaml:2]: no longer maintained

File: config/jobs/kubeflow/kubeflow-presubmits.yaml [hunk 6]
```diff
@@ -55,33 +27,6 @@ presubmits:
       testgrid-dashboards: sig-big-data
       description: Presubmit tests for Kubeflow.
       testgrid-num-columns-recent: '30'
-  kubeflow/website:
```
Comment (Bobgy) [inline config/jobs/kubeflow/kubeflow-presubmits.yaml:58]: empty test or always skipped optional test

**Review by chaodaiG — APPROVED** [review]

**Pull request merged**